yarn-project/end-to-end/src/e2e_account_contracts.test.ts

```diff
@@ -62,7 +62,7 @@ function itShouldBehaveLikeAnAccountContract(getAccountContract: (encryptionKey:
       ).getWallet();
       const childWithInvalidWallet = await ChildContract.at(child.address, invalidWallet);
       await expect(childWithInvalidWallet.methods.value(42).simulate()).rejects.toThrowError(
-        /could not satisfy all constraints/,
```

Comment:
hmm why the change?

Reply:
honestly not sure :grimacing:

Reply:
Seems like something that shouldn't just have happened in master too....?

Reply:
I see a change here from Alvaro today, just chock it up to getting ahead of noir bumping ACVM I guess